docs/guides/security.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Security"
+linkTitle: "Security"
+weight: 10
+slug: security
+---
+
+Cortex must be deployed with due care over system configuration, using principles such as "least privilege" to limit any exposure due to flaws in the source code.
+
+You must configure authorisation and authentication externally to Cortex; see [this guide](auth)
```

Comment:
I'm not sure this work after compiling the website. We should link the path to the specific `.md` file, this way the link is also checked by the compiler:

```suggestion
You must configure authorisation and authentication externally to Cortex; see [this guide](./authentication-and-authorisation.md)
```